Editor/foleys_GUITreeEditor.h

```diff
@@ -40,10 +40,37 @@ class GUITreeEditor  : public juce::Component,
 public:
     GUITreeEditor (MagicBuilder& builder);
 
-    void paint (juce::Graphics&) override;
     void resized() override;
 
+    void setValueTree (juce::ValueTree& refTree);
+
+    std::function<void (juce::ValueTree&)> onSelectionChanged {nullptr};
```

Comment:
std::function is implicitly created as nullptr, no need to add the default argument.
I am on the fence with that idea, will have to think about...